### PR TITLE
docs: add CI testing story

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: docs/documentation/stories.md [hunk 1]
```diff
@@ -29,3 +29,4 @@
  - [Deploy to GitHub Pages](stories/github-pages)
  - [Linked Library](stories/linked-library)
  - [Multiple apps](stories/multiple-apps)
+ - [Automated testing](stories/automated-testing)
```
Comment (hansl) [inline docs/documentation/stories.md]: Could we name it `Continuous Integration`? Someone looking for travis instructions is more likely to find CI than "Automated Testing".

File: docs/documentation/stories/automated-testing.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+Your project should start building.
+
+Be sure to check out the [Circle CI docs](https://circleci.com/docs/2.0/) if you want to know more.
+
+
+## Using Travis CI
+
+Create a folder called `.travis.yml` at the project root:
+
+```
```
Comment (hansl) [inline docs/documentation/stories/automated-testing.md]: Format this as yaml

File: docs/documentation/stories/automated-testing.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+For CI environments it's also a good idea to disable progress reporting (via `--no-progress`)
+to avoid spamming the server log with progress messages.
+
+
+## Using Circle CI
+
+Create a folder called `.circleci` at the project root, and inside of it create a file called
+`config.yml`:
+
+```
```
Comment (hansl) [inline docs/documentation/stories/automated-testing.md]: Format this as yaml.

Comment (filipesilva) [inline docs/documentation/stories/automated-testing.md]: Fixed

File: docs/documentation/stories/automated-testing.md (hunk 2 above)
Comment (filipesilva) [inline docs/documentation/stories/automated-testing.md]: Fixed

File: docs/documentation/stories.md (hunk 1 above)
Comment (filipesilva) [inline docs/documentation/stories.md]: Fixed

File: docs/documentation/stories/continuous-integration.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+Next you'll need to [sign up for Circle CI](https://circleci.com/docs/2.0/first-steps/) and
+[add your project](https://circleci.com/add-projects).
+Your project should start building.
+
+Be sure to check out the [Circle CI docs](https://circleci.com/docs/2.0/) if you want to know more.
+
+
+## Using Travis CI
+
+Create a folder called `.travis.yml` at the project root:
```
Comment (filipesilva) [inline docs/documentation/stories/continuous-integration.md]: Fixed 👍

**Pull request merged**